packages/docusaurus-theme-classic/src/theme/CodeBlock/styles.module.css

```diff
@@ -30,7 +49,11 @@
     bottom 200ms ease-in-out;
 }
 
-.codeBlock:hover > .copyButton {
+.copyButtonWithTitle {
+  top: calc(32px + var(--ifm-pre-padding))!important;
```

Comment:
`!important`  is definitely needed here?

Reply:
No. It works without `!important`. I added it just for emphasizing to override `copyButton` class with `copyButtonWithTitle`.